### ISSUE TITLE
[BUG/ISSUE] Incorrect rates in stratosphere for HOBr + SO3/HSO3

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#138]
Seb Eastham originally reported this issue.  I opened this Github issue so that we can track this better.

>While implementing code to turn off the reactions which GMI found to be problematic in the stratosphere, I came across what I believe is a bug in the heterogeneous chemistry code. The new HOBr + SO3 and HOBr + HSO3 heterogeneous chemistry appears to be intended to take place on tropospheric liquid and ice clouds. However, it looks like whoever implemented these reactions copied the HOBr + HCl code. This applies pre-calculated sticking coefficients for stratospheric sulfates and PSCs which are only appropriate to HOBr + HCl, but are now being incorrectly used in HOBr + SO3 and HOBr + HSO3. I’d like to recommend that the rate coefficient is zeroed out for stratospheric aerosols in the HetHOBr_HSO3 and HetHOBr_SO3 functions (specifically, we can just remove the lines

```F90
         ELSEIF (LUCX_BrCl.and.(N.eq.13)) THEN

            ! SSA/STS
            XSTKCF = KHETI_SLA(10)

         ELSEIF (LUCX_BrCl.and.(N.eq.14)) THEN

            ! Ice/NAT PSC
            IF (NATSURFACE) THEN
               XSTKCF = 0.1e+0_fp ! NAT
            ELSE
               XSTKCF = 0.3e+0_fp ! Ice
            ENDIF
```
>in only the HetHOBr_SO3 and HetHOBr_HSO3 code). I’m about 95% sure that these lines were included in error.

**Comment by yantosca** [comment]
Attaching some email conversations to this issue:

From: Becky Alexander <beckya@uw.edu> 

>Does this error and the suggested fix only apply to the stratosphere, or will it also impact trop chem?

From Seb Eastham <seastham@mit.edu>

>I believe this should only affect the stratosphere – it basically fixes a problem where the wrong rate is being applied for stratospheric aerosols only.

**Comment by yantosca** [comment]
Adding this comment:
> I chatted with Qianjie Chen (the original code author) about this yesterday and we agree with your suggested fix.  

We will add this fix into 12.8.0.

**Comment by yantosca** [comment]
Now slated for 12.9.0

**Comment by WilliamDowns** [comment]
This issue is fixed / overridden in 12.9.0 by updates to halogen chemistry from Xuan Wang.
